index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <header class="top-bar">
+        <a href="#" class="logo">My logo</a>
+        <nav class="menu"><a class="menu-icon" href=""><img src="images/menu.png" alt=""></a></nav> 
+        </header>
+        <div class="image-icons">
+            <ul>
+            <li><div class="left-ellipse"><img src="images/Ellipse11.png" alt=""></div></li>
+            <li><div class="center-ellipse1"><img src="images/Ellipse5.png" alt=""></div></li>
+            <li><div class="center-rectangle"><img class="center-rect" src="images/Rectangle7.png" alt=""></div></li>
+            <li><div class="center-ellipse2"><img src="images/Ellipse08.png" alt=""></div></li>
+            <li><div class="right-ellipse"><img src="images/Ellipse3.png" alt=""></div></li>
+            <li><div class="right-rectangle"><img src="images/Rectangle4.png" alt=""></div></li>
+        </ul>
+        </div>
+        <main>
+        <div class="text"><h1>I'm Rehan <br>glade to see <br>you!</h1>
+            <p class="paragraph">I’m a software developer! I can help you <br> build a product , feature or website Look <br> through some of my work and <br> experience! If you like what you see and <br>  have a project you need coded, don’t <br> hestiate to contact me.</p>
+            </div>
+            <div class="social-contact">
+                <h2 class="header">LET'S CONNECT</h2>
+                <a href=""><img class="social-icons" src="images/twitter.png" alt=""></a>
+                <a href=""><img class="social-icons" src="images/medium.png" alt=""></a>
+                <a href=""><img class="social-icons" src="images/github.png" alt=""></a>
+                <a href=""><img class="social-icons" src="images/angelist.png" alt=""></a>
+                <a href=""><img class="social-icons" src="images/angellist.png" alt=""></a>
+            </div>
+        </main>
+        <div class="circle-bottom"><img src="images/Ellipse15.png" alt=""></div>
+        <div class="ellipse-bottom-center"><img src="images/Ellipse4.png" alt=""></div>
+        <div class="ellipse-bottom-right"><img src="images/Ellipse12.png" alt=""></div>
+    
+</body>
+</html>
+
```

Comment:
- Please try to  align the elements according to the requirement, you should use Flexbox in all 2 sections(toolbar and headline section.)

- Both toolbar and headline section must have the margins (top, bottom, left, right) defined in the Template design.
-    You must stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1)

Your project looks like this
![Backgr](https://user-images.githubusercontent.com/93598189/185227809-6791d511-dfd2-4dc3-abb8-899fa7d39825.PNG)

The Figma design looks like :

![Figma design](https://user-images.githubusercontent.com/93598189/185227877-12c9398c-c2e2-4a87-82ed-bb34053f2750.PNG)